### ISSUE TITLE
Linear-Gradient with Angle (0deg) not rendering 

### ISSUE THREAD

**Issue by nammu** [niklasvh/html2canvas#172]
Hi !

First of all, nice plugin here ! I just needed something like that ;)

I just have a question about gradient.

Gradients are rendered when the angle is specified with a word (Top, Left, ...), but not when I put an angle in Degree (For exemple : linear-gradient(0deg, rgb(0, 0, 0) 0%, rgb(255, 255, 255) 100%) won't work).

Is there a way to fix this ? Because I use a lot of gradients with degrees values :)

Thanks !

**Comment by NicolayThafvelin** [comment]
There is a  simple fix for this just add this `if` before existing the position switch in the LinearGradientContainer
```javascript 
if (imageData.args[0].indexOf('deg') != -1) {
  var rad = parseFloat(imageData.args[0].substr(0, imageData.args[0].length - 3)) * (Math.PI / 180);
  //Finds y start and scales it between 0 and 1
  this.y0 = (Math.cos(rad) + 1) / 2;
  //Flips y1
  this.y1 = 1 - this.y0;
  //Same as for y0 but flip axis to match with css gradient
  this.x0 = (-Math.sin(rad) + 1) / 2;
  this.x1 = 1 - this.x0;
} else {
  imageData.args[0].split(" ").reverse().forEach(function(position) {
     switch (position) {
          ........
     }
}
```

**Comment by usmonster** [comment]
@NicolayThafvelin Pull requests are welcome... :]

**Comment by topwood** [comment]
@NicolayThafvelin  3q

**Comment by niklasvh** [comment]
Fixed in 1.0.0
